.github/workflows/deploy_stable_appstore.yml

```
@@ -0,0 +1,85 @@
+
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Ruby v3
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.3.9
+          bundler-cache: true
+
+      - name: Setup Rust and Cargo
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.12)

[error] the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)